great_expectations/rule_based_profiler/estimators/quantiles_numeric_range_estimator.py

```diff
@@ -68,8 +70,20 @@ def _get_numeric_range_estimate(
                 DEFAULT_QUANTILES_QUANTILE_STATISTIC_INTERPOLATION_METHOD
             )
 
+        datetime_detected: bool = datetime_semantic_domain_type(domain=domain)
+        metric_values_converted: np.ndaarray
+        (
+            _,
+            _,
```

Comment:
Same question. If conversion failed, don't we want to know that?

Reply:
@NathanFarmer They are not used in the present function -- only the converted array (which, since it is best effort, will be accepted with whatever `dtype` it will carry).  Thanks!